src/routes/api/db-migrations/db-migration-utils.ts

```diff
@@ -0,0 +1,90 @@
+import { Request } from "express";
+import fs from "fs";
+import path from "path";
+import { isNodeDev, isNodeTest, isNodeProd, getNodeEnv } from "utils/is-node-env";
+import { getLogger } from "config/logger";
+import { exec as execOrigin } from "child_process";
+import { promisify } from "util";
+
+const exec = promisify(execOrigin);
+const logger = getLogger("DBMigration");
+
+/**
+ * Return the common param in req.body `targetScript` to migrate up or rollback
```

Comment:
/nit `@return` - I think the convention is to use java-like doc

Reply:
Why "migrate up" only? What about "down"?

Reply:
fixed. `rollback` means `down`. But anyway I converted all `rollback` to `down` to be consistent.